src/modules/utils/panel/screens/MainMenuScreen.cpp

```diff
@@ -82,7 +82,7 @@ void MainMenuScreen::setupConfigureScreen()
     mvs->addMenuItem("Z Home Ofs",
         []() -> float { void *rd; PublicData::get_value( endstops_checksum, home_offset_checksum, &rd ); return rd==nullptr ? 0.0F : ((float*)rd)[2]; },
         [this](float v) { send_gcode("M206", 'Z', v); },
-        0.1F
+        0.01F
```

Comment:
I agree that it is unrelated and I would be happy to put in separate pull request. But why would it not be accepted? ```Z Home Ofs``` allows adjusting bed offset from the panel. Having an accuracy of 0.1 mm makes it unusable for that purpose.